### PR TITLE
LRQA-72181 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-72181

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase [hunk 1]
```diff
@@ -82,15 +82,14 @@ definition {
...
 			AssertTextEquals(
-				key_alertMessage = "Please enter no more than 100 characters.",
-				locator1 = "Message#ERROR_ENTER_A_VALID_VALUE",
-				value1 = "${key_alertMessage}");
+				locator1 = "TextInput#NAME",
+				value1 = "Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor sit amet Lorem ipsum dolor s");
 
-			Accounts.openAccountsAdmin();
-
-			LexiconEntry.viewNoEntry(rowEntry = "${accountName}");
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase:93]: @melodywu can we still do the assertion in Accounts Admin? So we can make sure the account wasn't saved with the long name?

Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase:93]: Done thanks

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
@melodywu I updated the description too since the behavior has changed. I also made a couple of refactor. See https://github.com/patriciajeanperez/liferay-portal/pull/534.

**Pull request closed without merging**